tasks/main.yml

```diff
@@ -1,19 +1,15 @@
 ---
 - name: Include OS-specific variables.
-  include_vars: "{{ ansible_os_family }}.yml"
+  include_tasks: variables.yml
 
 - name: Ensure SVN packages are installed.
   yum: "name={{ item }} state=present"
-  with_items:
-    - mod_dav_svn
-    - subversion
+  with_items: "{{ svn_packages }}"
```

Comment:
Note that for better efficiency, we can rewrite this task to:

```
yum: "name={{ svn_packages }} state=present"
```

Reply:
Interesting. How is the inline syntax more efficient?

Reply:
@aidanmelen apparently the way the internals of the module work, it will run _one_ yum command over the wire if you pass the array directly to `name`, but it will run multiple yum commands if you use `with_items`. It's a small speedup in this case, but it's worth it and makes the task simpler too :)

Reply:
That makes sense, Thanks.